services/web/server/tests/integration/01/test_project_workflow.py

```diff
@@ -113,7 +113,7 @@ async def _mock_copy_data_from_project(*args):
     # requests storage to delete data
     # mock1 = mocker.patch('simcore_service_webserver.projects.projects_handlers.delete_data_folders_of_project', return_value=None)
     mock1 = mocker.patch(
-        "simcore_service_webserver.projects.projects_handlers.projects_api.delete_data_folders_of_project",
+        "simcore_service_webserver.projects._delete.delete_data_folders_of_project",

```

Comment:
hmm.. you might need to rename these. I also changed them in my PR

Reply:
yes, i saw the conflict :-D All good!